docs/contributing/language_guide.rst

```diff
@@ -0,0 +1,60 @@
+Language Guide
+==============
+
+Purpose
+-------
+
+This guide defines and explains Pulp terms and concepts. Before you contribute to the Pulp docs,
+please read through this guide and use the established language where possible. This document is
+designed to be comprehensive, and should be understandable to an anyone who is familiar with
+software management. This document favors clarity and accuracy over ease of reading.
+
+Notation
+--------
+
+Each paragraph is preceded by a list of terms that are covered for the first time. In the paragraph
+body, terms that correspond to Python code are capitalized based on their type (see PEP 8) and are
+indicated by double backticks \`\`object\`\`. (``ClassName``, ``function_name``).  Terms that are
+not code objects in in bold. (**plugin**)
+
+Definitions
+-----------
+
+[``ContentUnit``, **plugin**, ``type``, ``Artifact``]
+    ``pulpcore`` is a generalized backend with a REST API and a plugin API. You will also need at
+    least one **plugin** to manage content.  Each **plugin** defines at least one ``type`` of
+    ``ContentUnit`` (like .rpm or .deb), which is the smallest set of data that can be managed by
+    Pulp. (The plural form of ``ContentUnit`` is a ``ContentUnits``, rather than Content or Units).
+    Files that belong to a ``ContentUnit`` are called ``Artifacts``, and each ``ContentUnit`` can
+    have 0 or many ``Artifacts``.  ``ContentUnits`` can share ``Artifacts``.
+
+[``Repository``, **add**, **remove**, **RepoVersion**]
+    ``ContentUnits`` in Pulp are organized by their membership in ``Repositories``. You can **add**
```

Comment:
The first two sentences need updating with repo versions.

Reply:
I changed this a little bit, (added "over time"), but that change doesn't address your concerns. My thinking was that we will still talk about "adding ContentUnits to a "Repository", and that RepositoryVersions are /how/ that is done.